### ISSUE TITLE
Test-task called from watch never finishes if a test lacks end() or (correct) plan() call

### ISSUE THREAD

**Issue by Zache** [yuanqing/gulp-tape#13]
The watch

``` javascript
gulp.task('watch', function() {
  return gulp.watch('fixtures/test/*.js', ['test']);
});
```

A test

``` javascript
test('planned', function(t) {
  // t.plan(1); // works
  // t.plan(2); // test-task called from watch never completes
  t.equal(abs(1), 1);
  // t.end(); // works like t.plan(1), use neither and the test-task never finishes
});
```

The errors are correctly reported if run outside of the watch.

**Comment by zoubin** [comment]
Tape listens to the `exit` event fired on `process` to check if all tests end.
But in watch, the process never exits.
If we are going to fix this, we have to run each file in a subprocess.
How do you think about that? @yuanqing

**Comment by yuanqing** [comment]
Closing because this issue should be fixed as of `0.0.10`
